cppfd/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.10)
+
+# Set the project name
+project(main)
+
+# Find required dependencies
+find_package(Kokkos)
+include_directories(/usr/local/include)
```

Comment:
This is ok as a hack, but you shouldn't have to do this.

Reply:
Thanks @lifflander, but I cannot get the clang compiler to look at this directory for includes. Is this a CMake variable that I should add at configuration time instead?

Reply:
@lifflander any idea why this seems to be needed to get the code to compile? I tried to remove it but it failed.

Reply:
When I remove this on my system, it works perfectly fine. Can you show me the exact error you are getting?

Reply:
It is now working without this line, it might have to do with how I installed Kokkos.